app/models/bus_stop.rb

```diff
@@ -163,10 +163,15 @@ class BusStop < ApplicationRecord
 
   LIMITED_ATTRS = {
     name: 'Stop Name',
+    id: 'Id',
     hastus_id: 'Hastus ID',
     route_list: 'Routes',
-    created_at: 'Created',
-    updated_at: 'Last updated'
+    created_at: 'Created at',
+    updated_at: 'Last updated',
+    updated_by: 'Last updated by',
```

Comment:
Why is what's listed here not the actual name of the attribute? The effect of which is that you just have to remap it below.

Reply:
There is no `updated_by` attribute, and the others were just added to `LIMITED_ATTRS` for the sake of order. If that's what you're asking?

Reply:
Yes, I am asking why you did that. Why not call it `last_updated_by` and then remove the `if` statement below?